substrate/frame/assets/src/weights.rs

```diff
@@ -530,6 +531,9 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(2_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
+	fn revoke_all_privileges() -> Weight {
+		Weight::zero()
```

Comment:
```suggestion
		Weight::MAX
```

Safer default

Reply:
yes, I now ran benchmarks on my machine for a better default